rules/js.txt

```diff
@@ -2426,6 +2480,7 @@
 !extends FontFaceSetLoadEvent Event
 <new Event> = <FontFaceSetLoadEvent>;
 <new sequence_FontFace_> = <FontFaceSetLoadEvent>.fontfaces;
+<new sequence_FontFace_> = new Array(<FontFace>);
```

Comment:
Nit: This line should be probably go above under FontFace, not under FontFaceSetLoadEvent object.

Reply:
Fixed